server/config/config.exs

```diff
@@ -7,6 +7,11 @@
 # General application configuration
 use Mix.Config
 
+# This prevents the GRPC server being automatically started when you start the app,
+# the reason we dont want it is since if this happens, you'll not be able to run ```iex -S mix``` since
+# it'll also try to start the GRPC server and fail on port being already used(duh)
+config :grpc, start_server: false
```

Comment:
maybe lets start it automatically on dev?

Reply:
IMO its annoying, i started with that.. but then if you do a ‘iex -S mix’ it fails since it tries to start the server again